lib/tpm2.h

```diff
@@ -277,4 +288,45 @@ tool_rc tpm2_quote(ESYS_CONTEXT *esys_context, tpm2_loaded_object *quote_obj,
 tool_rc tpm2_unseal(ESYS_CONTEXT *esys_context, tpm2_loaded_object *sealkey_obj,
         TPM2B_SENSITIVE_DATA **out_data);
 
+tool_rc tpm2_incrementalselftest(ESYS_CONTEXT *ectx, const TPML_ALG *toTest,
+        TPML_ALG **toDoList);
+
+tool_rc tpm2_stirrandom(ESYS_CONTEXT *ectx, const TPM2B_SENSITIVE_DATA *inData);
+
+tool_rc tpm2_selftest(ESYS_CONTEXT *ectx, TPMI_YES_NO fullTest);
+
+tool_rc tpm2_gettestresult(ESYS_CONTEXT *ectx, TPM2B_MAX_BUFFER **outData,
+        TPM2_RC *testResult);
+
+tool_rc tpm2_loadexternal(ESYS_CONTEXT *ectx,
+        const TPM2B_SENSITIVE *inPrivate,
+        const TPM2B_PUBLIC *inPublic,
+        TPMI_RH_HIERARCHY hierarchy,
+        ESYS_TR *objectHandle);
+
+tool_rc tpm2_pcr_event(ESYS_CONTEXT *ectx,
```

Comment:
every function argument needn't be on the newline, collapse this to character width per coding guideline.